build.sbt

```diff
@@ -179,21 +210,21 @@ lazy val examples = crossProject(JSPlatform, JVMPlatform)
       "org.http4s" %%% "http4s-ember-client" % http4sVersion,
       "org.tpolecat" %%% "natchez-xray" % natchezVersion,
       "org.tpolecat" %%% "natchez-http4s" % "0.6.0",
-      "org.tpolecat" %%% "skunk-core" % "0.6.4"
+      "org.tpolecat" %%% "skunk-core" % "0.6.3"
```

Comment:
Why is this rolled back?

Reply:
The answer to this question is lost to time

If there was a reason I've forgotten, but it's probably a mistake